Workbooks/Virtual Machines - Performance Analysis/Performance Analysis for a Single VM/Performance Analysis for a Single VM.workbook

```diff
@@ -248,363 +211,438 @@
       "type": 3,
       "content": {
         "version": "KqlItem/1.0",
-        "query": "let grain = totimespan('{grain}');\r\nlet windowsNetwork=Perf\r\n| where TimeGenerated {TimeRange}\r\n| where Computer == '{Computer}'\r\n| where ObjectName == 'Network Adapter' and CounterName == 'Bytes Sent/sec'\r\n| summarize CounterValue = sum(CounterValue) by bin(TimeGenerated, 2s);\r\nlet linuxNetwork=Perf\r\n| where TimeGenerated {TimeRange}\r\n| where Computer == '{Computer}'\r\n| where ObjectName == 'Network' and CounterName == 'Total Bytes Transmitted'\r\n| order by InstanceName, TimeGenerated asc\r\n| extend prev_Value=prev(CounterValue), prev_t=prev(TimeGenerated), prev_instance=prev(InstanceName)\r\n| project TimeGenerated, CounterValue=iff(prev_instance == InstanceName and CounterValue >= prev_Value and TimeGenerated > prev_t, (CounterValue - prev_Value) / ((TimeGenerated - prev_t) / 1s), real(0))\r\n| summarize CounterValue = sum(CounterValue) by bin(TimeGenerated, 2s);\r\nlet networkDataSend = union windowsNetwork, linuxNetwork;\r\nnetworkDataSend\r\n| summarize AVERAGE = round(avg(CounterValue), 2) by bin(TimeGenerated, grain)",
+        "query": "let grain = totimespan('{grain}');\r\nlet windowsNetwork=Perf\r\n| where TimeGenerated {TimeRange}\r\n| where Computer == Computer\r\n| where ObjectName == 'Network Adapter' and CounterName == 'Bytes Sent/sec'\r\n| summarize CounterValue = sum(CounterValue) by bin(TimeGenerated, 2s);\r\nlet linuxNetwork=Perf\r\n| where TimeGenerated {TimeRange}\r\n| where Computer == Computer\r\n| where ObjectName == 'Network' and CounterName == 'Total Bytes Transmitted'\r\n| order by InstanceName, TimeGenerated asc\r\n| extend prev_Value=prev(CounterValue), prev_t=prev(TimeGenerated), prev_instance=prev(InstanceName)\r\n| project TimeGenerated, CounterValue=iff(prev_instance == InstanceName and CounterValue >= prev_Value and TimeGenerated > prev_t, (CounterValue - prev_Value) / ((TimeGenerated - prev_t) / 1s), real(0))\r\n| summarize CounterValue = sum(CounterValue) by bin(TimeGenerated, 2s);\r\nlet networkDataSend = union windowsNetwork, linuxNetwork;\r\nnetworkDataSend\r\n| summarize {MetricsRight} by bin(TimeGenerated, grain)",
         "showQuery": false,
         "size": 0,
         "aggregation": 3,
         "showAnnotations": true,
         "showAnalytics": false,
         "timeContextFromParameter": null,
-        "resourceType": "microsoft.operationalinsights/workspaces",
+        "resourceType": "microsoft.compute/virtualmachines",
         "crossComponentResources": [
-          "{Workspace}"
+          "{Computer}"
         ],
         "visualization": "timechart"
       },
-      "conditionalVisibility": {
-        "parameterName": "Computer",
-        "comparison": "isNotEqualTo",
-        "value": null
-      }
+      "customWidth": "50"
     },
     {
       "type": 1,
       "content": {
-        "json": "### Bytes Received Rate B/s ({Computer})"
+        "json": "### Bytes Received Rate B/s"
```

Comment:
if this is the last metric, it might look better if it fills the width of the page or else there's blank space on the right

Reply:
Hello Vu, I can add a description within the workbook to explain what is going on. As for the last chart, I wanted to keep it at 50% to closely mirror what the user sees on the Single VM Performance view. Also it would be inconsistent if the Bytes Received chart was wider than the Bytes Sent chart, but I'll leave this up to you. Thank you.

Reply:
@vutran01 I have added description, I still left the last chart at 50% width per comment above.